nameable.rb

```diff
@@ -0,0 +1,8 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+  end
+end
+
+# n = Nameable.new()
+# p n.correct_name
```

Comment:
- [Optional] Kindly remove all the commented lines of code rom your codebase, this will make your codebase to be cleaner and readable. ℹ️